SingularityUI/app/components/requestDetail/RequestUtilization.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import CollapsableSection from '../common/CollapsableSection';
+import { UsageInfo } from '../common/statelessComponents';
+import Utils from '../../utils';
+import BootstrapTable from 'react-bootstrap/lib/Table';
+
+const RequestUtilization = ({isFetching, utilization}) => {
+  const isCpuOverAllocated = utilization &&
+    (utilization.maxCpuUsed > utilization.cpuReserved / utilization.numTasks);
+  const attributes = utilization && (
+      <div className="row">
+        <div className="col-md-3">
+          <UsageInfo
+            title="CPU per task average"
+            total={utilization.cpuReserved / utilization.numTasks}
+            used={utilization.cpuUsed / utilization.numTasks}
+            style={utilization.cpuUsed >= utilization.cpuReserved ? 'danger' : null}
+          >
+            <p>{Utils.roundTo(utilization.cpuUsed / utilization.numTasks, 2)} of {utilization.cpuReserved / utilization.numTasks} CPU reserved</p>
```

Comment:
Move these percentage calculations into a separate utility function instead of using the magic number `2` all over the place.